src/main/java/com/scg/stop/domain/project/dto/response/ProjectResponse.java

```diff
@@ -0,0 +1,45 @@
+package com.scg.stop.domain.project.dto.response;
+
+import static lombok.AccessLevel.PRIVATE;
+
+import com.scg.stop.domain.project.domain.Member;
+import com.scg.stop.domain.project.domain.Project;
+import com.scg.stop.domain.project.domain.ProjectCategory;
+import com.scg.stop.domain.project.domain.ProjectType;
+import lombok.AllArgsConstructor;
+import lombok.Builder;
+import lombok.Getter;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+@Getter
+@Builder
+@AllArgsConstructor(access = PRIVATE)
+public class ProjectResponse {
+    private String thumbnailUrl;
+    private String projectName;
+    private String teamName;
+    private List<String> studentsName;
+    private String professorName;
+    private ProjectType projectType;
+    private ProjectCategory projectCategory;
```

Comment:
project가 클래스 이름에 이미 포함이 되어있어서 아래와 같이 수정하는건 어떨지 제안해봅니다

```suggestion
private ProjectType type;
private ProjectCategory category;
```

Reply:
이부분은 api 반환 json의 key값으로 적용이 되는 부분이라서, 프론트엔드에서 헷갈리지 않도록 하는 목적이 있긴 했는데, 어떻게 생각하시나요??

Reply:
아하 그런 의도로 생각하면 기존 방법이 좋은 것 같습니다 👍

Reply:
넵 감사합니다!

Reply:
054bb93